pyaedt/misc/console_setup

```diff
@@ -8,9 +8,17 @@ cpython_console.py should be run instead of this script.
 This file can also serve as a template to modify PyAEDT scripts to take advantage of the command line arguments
 provided by the launcher
 """
+import os
 import sys
 import atexit
-from pyaedt import *
+try:
+   from pyaedt import *
+except ImportError:
+   # debug only
+   console_path = os.path.dirname(__file__)
+   if "PersonalLib" in console_path:
+      sys.path.append(os.path.join(console_path, "..", "..", ".."))
```

Comment:
@maxcapodi78  I think a comment here would be helpful on what we are trying to achieve - and what are we trying to apped to path if the file is in PersonalLib

Reply:
added